contracts/PolyDistribution.sol

```diff
@@ -17,14 +17,14 @@ contract PolyDistribution is Ownable {
 
   uint256 private constant decimalFactor = 10**uint256(18);
   enum AllocationType { PRESALE, FOUNDER, AIRDROP, ADVISOR, BONUS, RESERVE }
+  uint256 public constant INITIAL_SUPPLY   = 1000000000 * decimalFactor;
   uint256 public AVAILABLE_TOTAL_SUPPLY    = 1000000000 * decimalFactor;
   uint256 public AVAILABLE_PRESALE_SUPPLY  = 240000000 * decimalFactor; // 100% Released at Token Distribution (TD)
   uint256 public AVAILABLE_FOUNDER_SUPPLY  = 150000000 * decimalFactor; // 25% Released at TD +1 year -> 100% at TD +4 years
   uint256 public AVAILABLE_AIRDROP_SUPPLY  = 10000000 * decimalFactor;  // 100% Released at TD
   uint256 public AVAILABLE_ADVISOR_SUPPLY  = 25000000 * decimalFactor;  // 100% Released at TD +7 months
   uint256 public AVAILABLE_BONUS_SUPPLY    = 80000000 * decimalFactor;  // 25% Released at TD +1 year -> 100% at TD +4 years
   uint256 public AVAILABLE_RESERVE_SUPPLY  = 495000000 * decimalFactor; // 12.5% Released at TD +6 months -> 100% at TD +4 years
-  uint256 public grandTotalAllocated = 0;
```

Comment:
Decided to leave this one in because Shannon already created some cool analytics dashboards and didn't want to rebuild them for nothing.